src/com/owncloud/android/lib/resources/status/OCCapability.java

```diff
@@ -77,6 +78,7 @@ public OCCapability(){
         mVersionMicro = 0;
         mVersionString = "";
         mVersionString = "";
+        mServerName = "Nextcloud";
```

Comment:
@mario this is library code, ```R.string.app_name``` is part of the app project, so we need to use a "hardcoded value" here.